apps/Dataman/.expo/devices.json

```diff
@@ -0,0 +1,3 @@
+{
+  "devices": []
```

Comment:
what is this for?

Reply:
in this devices array we can define the device type and their configuration. may be useful in the future